### ISSUE TITLE
Kali linux package

### ISSUE THREAD

**Issue by noraj** [noraj/haiti#3]
New Tool Requests on Kali Bug Tracker: [ID 0006797](https://bugs.kali.org/view.php?id=6797)

**Comment by noraj** [comment]
Using [gem2deb](https://packages.debian.org/sid/gem2deb) ([git](https://salsa.debian.org/ruby-team/gem2deb)) should be easier than [pacur](https://github.com/pacur/pacur).

**Comment by Abhinandan-Khurana** [comment]
May I try homebrew?

**Comment by noraj** [comment]
@Abhinandan-Khurana Yes, please, I'm not a mac user so it's hard for me to test this. It will be greatly appreciated if you manage to do that.

**Comment by noraj** [comment]
@Abhinandan-Khurana Seems someone packaged it to homebrew already 4 days ago https://github.com/Homebrew/homebrew-core/commits/9debb24aaa764a45a351ad5e3f26381717dc0dc0/Formula/h/haiti.rb cf. https://formulae.brew.sh/formula/haiti

**Comment by Abhinandan-Khurana** [comment]
Ohh, Sure.
Then maybe I can work on snapcraft?

**Comment by noraj** [comment]
Please do.
